ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -100,12 +100,7 @@ else
    yamlpath=$GDASApp/parm/atm/obs/testing/${obtype}.yaml
 fi
 
-#exename=test_ObsFilters.x
```

Comment:
Why get rid of this here?

Reply:
For each data type we have two yaml files. One for HofX only (no data filtering) and one for HofX + QC.  
Original the test_ObsOperator.x is for the hofx only, and test_OBsFilters.x is for HofX+QC.  

However, I found that it is better include the error assignment part of the filter to the HofX only test for the following two reasons:
(1) the default EVA configuration will not failed due to missing observation errors
(2) we can validate the original (input) observation error with the GsiInput observation error stored in the obs file.  

Therefore, the test_ObsOperator.x was replaced with test_ObsOperator.x